SPECS/python-google-pasta/python-google-pasta.spec

```
@@ -0,0 +1,51 @@
+Summary:        Enable python source code refactoring through AST modifications
+Name:           python-google-pasta
+Version:        0.2.0
+Release:        1%{?dist}
+License:        ASLv2.0
```

Nit-pick: since we're now moving to SPDX identifiers, let's go with
```suggestion
License:        Apache-2.0
```